packages/node/src/config/index.js

```diff
@@ -1,16 +1,6 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-const defaultJson = require('./default.json');
-
-function getConfig() {
-  if (['localhost', 'testing'].includes(process.env.NODE_ENV)) {
-    // eslint-disable-next-line global-require, import/no-dynamic-require
-    const file = require(`./${process.env.NODE_ENV}.json`);
-    return Object.assign(defaultJson, file);
-  }
-  return defaultJson;
-}
-
 module.exports = {
-  ...getConfig(),
-  ...(process.env.METRICS_SERVER ? { host: process.env.METRICS_SERVER } : {}),
+  bufferLength: 1,
+  timeout: 2000,
+  host: process.env.README_METRICS_SERVER ? process.env.README_METRICS_SERVER : 'https://metrics.readme.io',
+  readmeApiUrl: 'https://dash.readme.io/api',
```

Comment:
We've never overridden this, and we mock out this URL in our integration tests so I'm not going to bother adding a `README_API_URL` env var to every SDK.